velox/exec/Driver.cpp

```diff
@@ -1090,6 +981,72 @@ folly::dynamic Driver::toJson() const {
   return obj;
 }
 
+template <typename Func>
+auto Driver::withDeltaCpuWallTimer(
```

Comment:
Do we need to return the timed function? Thanks!

Reply:
In the current cases no, but this allows withDeltaCpuWallTimer to time other operations that do return something.

Reply:
If not, let's keep it simple for now? Then runOpWithDeltaCpuWallTimer is bit straightforward than withDeltaCpuWallTimer.

Reply:
> If not, let's keep it simple for now? Then runOpWithDeltaCpuWallTimer is bit straightforward than withDeltaCpuWallTimer.

Actually I replied wrong. The timed operations can return the blocking reason. So it needs a return type.